src/DynamoCoreWpf/Extensions/ViewExtensionLoader.cs

```diff
@@ -60,6 +65,11 @@ public IViewExtension Load(string extensionPath)
                 {
                     definition.TypeName = item.InnerText;
                 }
+                else if (item.Name == "IsEnabled")
+                {
+                    // Because the default value for this property is true so we check if user specifically put it false
+                    definition.IsEnabled = !item.InnerText.Equals("false") && !item.InnerText.Equals("False");
```

Comment:
`bool.TryParse` could be used instead and only assign `definition.IsEnabled` if a boolean was actually parsed.

Reply:
Will do